go/internal/flag/flag_test.go

```diff
@@ -282,27 +282,16 @@ func TestArgs(t *testing.T) {
 }
 
 func TestIsZeroValue(t *testing.T) {
-	oldCommandLine := goflag.CommandLine
-
-	defer func() {
-		goflag.CommandLine = oldCommandLine
-	}()
-
 	var testFlag string
 
-	goflag.StringVar(&testFlag, "testflag", "default", "Description of testflag")
-	goflag.Parse()
+	testFlagSet := goflag.NewFlagSet("testFlagSet", goflag.ExitOnError)
+	testFlagSet.StringVar(&testFlag, "testflag", "default", "Description of testflag")
 
-	f := goflag.Lookup("testflag")
+	f := testFlagSet.Lookup("testflag")
 
-	// Test the isZeroValue function
 	result := isZeroValue(f, "")
-
 	assert.True(t, result)
 
-	// Test again with a non-zero value
 	result = isZeroValue(f, "anyValue")
-	if result {
-		t.Errorf("Expected false, got true. The value 'newvalue' should not be considered as zero value.")
-	}
+	assert.False(t, result)
```

Comment:
The one issue with `assert.True` and `False` is the diff messages aren't too useful on their own - they just read "expected true, got false" (or vice versa) and you then need to go back to the test to see what actually failed.

We should include failure messages when we use these particular methods so we don't need to do that.

Reply:
done.